client/js/lounge.js

```diff
@@ -1142,13 +1207,34 @@ $(function() {
 			var chan = $(this);
 			if (chan.find(".messages .msg:not(.unread-marker)").slice(0, -100).remove().length) {
 				chan.find(".show-more").addClass("show");
+
```

Comment:
I wish we got rid of this damn `setInterval`, not a fan of running this 'cleanup' code a lot.